packages/aws-cdk/lib/commands/flag-operations.ts

```diff
@@ -261,3 +297,86 @@ function formatTable(headers: string[], rows: string[][]): string {
   table += separator;
   return table;
 }
+
+export async function displayFlags(flagsData: FeatureFlag[], ioHelper: IoHelper, flagName?: string, all?: boolean): Promise<void> {
+  if (flagName && flagName.length > 0) {
+    const flag = flagsData.find(f => f.name === flagName);
+    if (!flag) {
+      await ioHelper.defaults.error('Flag not found.');
+      return;
+    }
+
+    await ioHelper.defaults.info(`Description: ${flag.explanation}`);
+    await ioHelper.defaults.info(`Recommended value: ${flag.recommendedValue}`);
+    await ioHelper.defaults.info(`User value: ${flag.userValue}`);
+    return;
+  }
+
+  const headers = ['Feature Flag Name', 'Recommended Value', 'User Value'];
+  const rows: string[][] = [];
+
+  const getFlagPriority = (flag: FeatureFlag): number => {
+    if (flag.userValue === undefined) {
+      return 3;
+    } else if (String(flag.userValue) === String(flag.recommendedValue)) {
```

Comment:
This condition appears multiple times throughout the code. Consider extracting it to a function.